docs/README.md

```diff
@@ -149,6 +149,9 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `productAssemblyOptionItemImage` |
 | `productAssemblyOptionItemName` |
 | `textInputValue` |
+| `inputValueOptionBoxItem` |
+| `inputValueOptionBoxItemActive` |
```

Comment:
This should be a modifier.

Use the `applyModifiers` function that is exported alongside the `useCssHandles` hook. Here's a [link to the docs](https://github.com/vtex-apps/css-handles/blob/7f8257d50a1d01977e4e06048790a0ea03003297/docs/README.md).

Reply:
done, please check againg